doc/src/modules/solvers/solveset.rst

```diff
@@ -398,7 +398,7 @@ How does ``solveset`` ensure that it is not returning any wrong solution?
     >>> from sympy import symbols, S, pprint, solveset
     >>> x, n = symbols('x, n')
     >>> pprint(solveset(abs(x) - n, x, domain=S.Reals), use_unicode=True)
-    [0, ∞) ∩ {n}
+    {x | x ∊ {-n, n} ∧ ({{n} ∈ [0, ∞)})}
```

Comment:
I don't follow what this means. For starters, it seems there are redundant parentheses (is there a bug in the printer?), but removing those, we get `x ∊ {-n, n} ∧ {...}`. The `{...}` bit is a set, which isn't a boolean predicate, so it doesn't make sense to have it in an `And`. Finally, `{n}` is a set whereas `[0, ∞)` is a set of numbers, so it will never be an element of it. You probably meant to use "subset", not "element of". Do we have a `Subset` object? 

The cleanest way to write this would be to use ForAll, but we don't have that yet (see https://github.com/sympy/sympy/pull/7608). A Subset would also work (it would just be less readable), as would an And over all the elements (which would be longer, but also less roundabout). 

Also, `∊` and `∈` are different characters (the former is "contains" and the latter is an epsilon).